wrapanapi/ec2.py

```diff
@@ -459,28 +459,60 @@ def set_name(self, instance_id, new_name):
     def get_name(self, instance_id):
         return self._get_instance(instance_id).tags.get('Name', instance_id)
 
-    def _get_item_for_tag_assigment(self, item_id, type):
-        if type == 'instances':
+    def _get_item_for_tag_assigment(self, item_id, item_type):
+        """Get ec2 item for tagging
+
+        Args:
+            item_id(str): instance/template name
+            item_type(str): pass ec2 item type, for now available only for instance and template
+        """
+        if item_type == 'instances':
             taggable_item = self._get_instance(item_id)
-        elif type == 'images':
+        elif item_type == 'images':
             taggable_item = self._get_image(item_id)
         else:
-            raise NotImplementedError("Get {} item method doesn't added".format(type))
+            raise NotImplementedError("Get {} item method doesn't added".format(item_type))
         return taggable_item
 
-    def set_tag(self, item_id, tag_label, tag_name, type='instances'):
```

Comment:
the rename from `type` to `item_type` looks like a unintended breaking api change

Reply:
yup, already corrected